src/pymorize/std_lib/exceptions.py

```diff
@@ -0,0 +1,20 @@
+"""
+This module contains custom exceptions that you should raise when something specific
+goes wrong in the standard library.
+"""
+
+# NOTE(PG): I am not sure what is better; to have this here, or to have each exception next to the step code...
```

Comment:
I don't know either. One one side it makes sense that exceptions that can be used everywhere live here, however, exceptions that are specific to some specific modules, is maybe better having them closer to the code... not a blocker though

Reply:
Let's just leave it here then.